tardis/montecarlo/montecarlo_numba/tests/test_base.py

```diff
@@ -80,15 +74,14 @@ def test_montecarlo_main_loop(
         actual_nu_bar_estimator, expected_nu_bar_estimator, rtol=1e-13
     )
     npt.assert_allclose(actual_j_estimator, expected_j_estimator, rtol=1e-13)
-    npt.assert_allclose(actual_energy.value, expected_energy, rtol=1e-13)
-    npt.assert_allclose(actual_nu.value, expected_nu, rtol=1e-13)
+    npt.assert_allclose(actual_energy, expected_energy, rtol=1e-13)
+    npt.assert_allclose(actual_nu, expected_nu, rtol=1e-13)
 
 
-@pytest.mark.xfail(reason="need to store virtual packet data in hdf5")
+@pytest.mark.xfail
```

Comment:
Is this test still xfail? Why?

Reply:
I added this tests. it will no longer be xfail with the syrupy update

Reply:
Can this xfail be removed then to make sure it passes?

Reply:
this is no longer xfail - it's outdated